### PR TITLE
Pull Request #2: Including Videos in Submissions Component

### PR DESCRIPTION
**Pull request by russell258**
Testing was done to check that video plays upon clicking

### CODE REVIEW

File: pages/submissions/[selectedSubmission].js [hunk 1]
```diff
@@ -27,13 +28,20 @@ const SelectedSubmission = () => {
...
 
     if (selectedSubmissionData) {
-        let images = <span className="flex flex-col flex-wrap items-start items-center w-12/12 md:items-start lg:flex-row lg:w-12/12 lg:items-stretch lg:justify-start">
-            {selectedSubmissionData.imagesUrl && selectedSubmissionData.imagesUrl.map(img => {
-                return (
-                    <img src={`${img}`} key={img} className="rounded-lg object-fill mb-3 w-12/12 md:w-5/6 lg:w-5/12 lg:mt-0 lg:mr-3 last-of-type:lg:mr-0" />
-                )
-            })}
-        </span>
+        console.log(selectedSubmissionData.videosUrl);
```
Comment (nevtay) [inline pages/submissions/[selectedSubmission].js:31]: Please remove console statements

**Comment by russell258** [comment]
**Summary**
1. Console log removed
2. Conditional rendering of placeholder text in submissions page when no videos url available
image


For your kind review, please

**Propose for upcoming changes**
- Scoping Positions into
a) Positions (Side Control, Back, Mount, Knee on Belly, North South)

While remaining "positions" should rightfully be categorized to "Guards".

Points of discussion for guards:
- Can segregate into Closed, Half, Open (Far Distance, Mid Distance, Near Distance)
- can expand into Guard defense, Guard offense, Guard Passing

There is a lot we can cover on this. For your comments.

File: pages/submissions/[selectedSubmission].js [hunk 2]
```diff
@@ -36,7 +36,26 @@ const SelectedSubmission = () => {
                     )
                 })}
             </span>
-
+        if (selectedSubmissionData.videosUrl.length===0){
```
[the remark this answers is not in the document]
Comment (russell258) [reply pages/submissions/[selectedSubmission].js]: **Summary**
1. Formatted equality spacing
2. Refactored selectedSubmissionData with ternary operator (thanks!)
3. Added Guards Page for your review
   --> It is following a blue theme, compared to yellow and purple for submissions and positions
4. Slight tidy-up of Positions (move closed guard out, renaming etc)

Main Page
image

Guards Page
image

Selected Guard Page
image

[the remark this answers is not in the document]
Comment (nevtay) [reply pages/submissions/[selectedSubmission].js]: LGTM!

**Pull request merged**